code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -932,3 +932,10 @@
 	results = list(/datum/reagent/consumable/ethanol/triumphal_arch = 10)
 	required_reagents = list(/datum/reagent/consumable/ethanol/mushi_kombucha = 5, /datum/reagent/consumable/ethanol/grappa = 2, /datum/reagent/consumable/lemonjuice = 2, /datum/reagent/gold = 1)
 	mix_message = "The mixture turns a deep golden hue."
+
+/datum/chemical_reaction/syndicate_screwdriver
+	name = "Syndicate Screwdriver"
+	id = /datum/reagent/consumable/ethanol/syndicate_screwdriver
+	results = list(/datum/reagent/consumable/ethanol/syndicate_screwdriver = 2)
```

Comment:
```suggestion
	results = list(/datum/reagent/consumable/ethanol/syndicate_screwdriver = 4)
```

Reply:
So just as stated in Discord, there are other drinks/chems that do this (morphine, bastion & hearty punch) which is I assume to stop the drink being easily spammed. I don't mind changing it so it makes sense but I also think, I will tweak it so it'll have the same as Hearty punch in terms of required reagents. 
(5 Syndicate bomb + 5 Screwdriver = 1 syndicate screwdriver)